identity-service/src/routes/healthCheck.js

```diff
@@ -269,6 +271,35 @@ module.exports = function (app) {
     }
   }))
 
+  app.get('/sol_balance_check', handleResponse(async (req, res) => {
+    let { minimumBalance } = req.query
+    minimumBalance = parseFloat(minimumBalance || config.get('solMinimumBalance'))
+    let solanaFeePayerWallet = config.get('solanaFeePayerWallet')
+
+    let solanaFeePayerWalletSet = false
+    let balance = 0
+
+    if (solanaFeePayerWallet) {
+      solanaFeePayerWalletSet = true
+      const publicKey = (new solanaWeb3.Account(solanaFeePayerWallet)).publicKey
```

Comment:
let's add the public wallet as part of the response? wdyt?

Reply:
Sounds good, adding now 👍